apps/calendar/js/views/month.js

```diff
@@ -105,12 +108,25 @@ Month.prototype = {
         this.controller.selectedDay = date;
         break;
       case 'dbltap':
-        this.app.go('/day/');
+        // make sure we discard double taps that started on a different day
+        if (this._lastTarget === target) {
+          this._goToAddEvent();
+        }
         break;
       case 'monthChange':
         this.changeDate(e.data[0]);
         break;
     }
+    this._lastTarget = target;
```

Comment:
it doesn't make any difference, `target` only exists for delegated events.. which in this case are only `dbltap` and `click`